docs/scripts-used-by-travis.rst

```diff
@@ -0,0 +1,7 @@
+Scripts used by Travis
+======================
+
+Some Travis build processes have a deploy component, and sometimes this fetches a script from this repository to run.
```

Comment:
"Some" makes me ask "which". Can we list them all? Is it just the standard?

Reply:
I'd question whether they would be kept fully up to date if there was a complete list here. Maybe we can make this one a issue and discuss that one further so as not to hold this up.